src/du/native/clonesize.c

```
@@ -1859,6 +1881,234 @@ void clonesize_free(char *p) { free(p); }
+    char *out = malloc(cap);
+    if (!out) { perror("malloc bigjson"); exit(1); }
+    size_t len = 0;
+    #define BEMIT(...) do { \
+        int need = snprintf(out + len, cap - len, __VA_ARGS__); \
+        if (need < 0) return out; \
+        if ((size_t)need >= cap - len) { cap = (cap + need) * 2; out = realloc(out, cap); \
+            if (!out) { perror("realloc bigjson"); exit(1); } \
+            need = snprintf(out + len, cap - len, __VA_ARGS__); } \
+        len += need; \
+    } while (0)
+    BEMIT("{\"files_listed\":%llu,\"dirs\":%llu,\"roots\":%d,\"min_bytes\":%llu,\"threads\":%d,\"walk_ms\":%llu,",
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)